### PR TITLE
With tag priorities we still should prefer released resources

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

**Comment by praiskup** [comment]
@FrostyX priority queue messed up "released" resource priorities :-/ can you take a look at this, please?

**Comment by FrostyX** [comment]
Such a clean solution, I like this.
+1

File: resallocserver/manager.py [hunk 1]
```diff
@@ -611,6 +615,12 @@ def _assign_tickets(self):
                         if tag.priority is not None and tag.id in ticket_tags:
                             priority += tag.priority
 
+                    if resource.sandbox:
+                        # Re-used resources should be preferred to avoid
+                        # allocating new and new resources for the same
+                        # sandboxes.  TODO, make this configurable once needed.
```
Comment (FrostyX) [inline resallocserver/manager.py:621]: I guess the TODO would be better above the `REUSED_RESOURCE_PRIORITY` definition but that's just nitpicking.

Comment (praiskup) [reply resallocserver/manager.py:621]: Indeed.

Comment (praiskup) [reply resallocserver/manager.py:621]: Done, and merged.  Thank you!

**Pull request closed without merging**